### PR TITLE
AC 538: Add Toast Message when going online.

### PR DESCRIPTION
**Pull request by Abdelaty**
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-538

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -338,6 +338,7 @@
     <!-- Syncbutton toasts -->
     <string name="disconn_server">Disconnecting from server</string>
     <string name="reconn_server">Reconnecting to server</string>
+    <string name="connected_to_server_message">Connected to internet successfully</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="connected_to_server_message">You are now connected!</string>
```

Comment (Abdelaty) [reply openmrs-client/src/main/res/values/strings.xml]: I did it, I change the message from connected to the server to "Connected to the internet successfully"

Comment (f4ww4z) [reply openmrs-client/src/main/res/values/strings.xml]: No, please change to my suggested change above.

**Comment by Abdelaty** [comment]
done

**Comment by Abdelaty** [comment]
@f4ww4z  can it be merged now?

**Pull request merged**